### PR TITLE
Snacks remade (also punpun and bones return) and tweaks to moebius

### PR DESCRIPTION
**Pull request by fernandos13**
## About The Pull Request
well as suggested from the #feedback channel in discord.
replaces snack vendors all around the ship. that and

medbay got 2 extra sinks

RnD now have 1

bones. and punpun are back at their houses

added a handmade pistol and some bullets to xenobio. maybe this i'll give em a better chance against slimes roundstart

added screwdrivers to fix synth organs in surgery

https://gyazo.com/collections/01819e163794e3fc0d80cce71f9188ea

## Why It's Good For The Game
to make bar and garden more meaningfull. since the sanity on itself didn't brought it to eris needs

also
adds sinks to medbay and robotics surger. for surgeons. to clean their paws
adds pets. cause cmon you gotta love em (and incentive to pick MBO and Bar manager)

## Changelog
🆑 Fernandos33
add: 2 sinks to moebius
add: handmade pistol and ammo to xenobio
add: 2 screwdrivers to surgery
del: snack vendors
tweak: seed servitor are in the same level as garden now. no more cardio needed
balance: some vendors got replaced with battery. cigarette and a soviet machine even.
/🆑

### CODE REVIEW

**Comment by fernandos13** [comment]
this again. hell how do i fix this shit once more? i tried opening and closing the DM edtor and yeah mapmerge is working as intended

File: maps/CEVEris/_CEV_Eris.dmm [hunk 1]
```diff
@@ -74165,6 +74186,11 @@
 /obj/machinery/atmospherics/pipe/simple/hidden/supply{
 	dir = 4
 	},
+/obj/item/weapon/gun/projectile/handmade_pistol{
+	step_x = 0;
```
Comment (Kurgis) [inline maps/CEVEris/_CEV_Eris.dmm:74190]: This breaks everything, remove the edited step vars.

Comment (Kurgis) [reply maps/CEVEris/_CEV_Eris.dmm:74190]: I suggest just editing the file directly, and deleting the two lines instead of using the map editor.

Comment (fernandos13) [reply maps/CEVEris/_CEV_Eris.dmm:74190]: oh thanks i'll do it right now

**Pull request merged**